functions/users.py

```diff
@@ -14,3 +13,23 @@ def disable_inactive_users(*args):
         for u in disabled:
             print(f"Disabled inactive: {u[0]}")
         print("done.")
+
+
+def refresh_download_permissions(*args):
+    """
+    Extend the expiry date for GCS download permissions belonging to users
+    who accessed the system today. If we don't do this, users whose accounts
+    are still active might lose GCS download permission prematurely.
+    """
+    active_today = lambda q: q.filter(
+        Users._accessed > datetime.today() - timedelta(days=1)
```

Comment:
nitpick: while pragmatically I think this should fly, I see a potential for a bug here, as in: proper semantic should update users accessed stuff since the last run. Current implementation I think might allow for some users to fall into the cracks between mis-scheduled runs or something. 
One possible easy solution is to increase time delta to cover multiple scheduling intervals (e.g. 2 days), which will lead to potentially unneeded updates in favor of losing a needed one.